components/cache/cache_pools.rst

```diff
@@ -127,9 +127,20 @@ when all items are successfully deleted)::
 .. tip::
 
     If the cache component is used inside a Symfony application, you can remove
-    *all items* from the *given pool(s)* using the following command (which resides within
+    items from cache pools using the following commands (which reside within
     the :ref:`framework bundle <framework-bundle-configuration>`):
 
+    To remove *one specific item* from the *given pool*:
+
+    .. code-block:: terminal
+
+        $ php bin/console cache:pool:delete <cache-pool-name> <cache-key-name>
```

Comment:
one thing that is not clear for me here is how you get the list of cache pool names :/

Reply:
You configured them yourself in the `framework.cache` section. Do you think we need to clarify that?

Reply:
I think so. Isn't there `app` and `system` as well by default?